docs/HowToUseIoTjs.md

```diff
@@ -0,0 +1,63 @@
+# How to use IoT.js
+
+
+IoT.js is an open source software platform
+for Internet of Things with JavaScript.
+
+More mode detailed information, please check:
+http://www.iotjs.net
+
+Or shipped [README](..//external/iotjs/README.md) in sources.
+
+
+## Use reference configuration
+
+   machine=artik053 # Adapt with your supported target
+   cd os && ./tools/configure.sh ${machine}/devel # Generate os/.config
+   make menuconfig # Verify WiFi credentials
+   make # Will build os image with IoT.js in and example
+  
+
+## To enable IoT.js feature
+
+Using menuconfig an example can be used to selected needed feature.
+
+   make menuconfig
+   # Application Configuration  --->
```

Comment:
Needs ``` block

Reply:
Or 4 spaces,

It's will be properly formatted:
https://github.com/TizenTeam/TizenRT/blob/sandbox/rzr/review/master/docs/HowToUseIoTjs.md

and easier to read in editor or terminal.